spec.md

```diff
@@ -100,8 +100,19 @@ This header allows brokers to reject requests from marketplaces for versions the
 
 ## Authentication
 
-The marketplace MUST authenticate with the service broker using HTTP
-basic authentication (the `Authorization:` header) on every request. The broker is responsible for validating the username and password and returning a `401 Unauthorized` message if credentials are invalid. It is RECOMMENDED that brokers support secure communication from platform marketplaces over TLS.
+While the communication between a platform and broker
+MAY be unsecure, it is RECOMMENDED that all communications between a
+platform and a broker are secured via TLS and authenticated.
+
+Unless there is some out of band communication and agreement between a
+platform and a service broker, the marketplace MUST authenticate with the
+service broker using HTTP basic authentication (the `Authorization:` header)
```

Comment:
@bparees Are you focused on the linking of basic-auth with the authZ header?  If so, I think that's ok for this paragraph because we're saying what the default is - basic auth via authZ header.  If you want to separate out the header name discussion from the header value discussion, can we do that? Can we really mandate that all auth mechanisms MUST use the Authorization header? I would think that it would be up to the spec of that mechanism to decide that.

If you're talking about something else then can you elaborate because I think your proposed text says the same thing as what's in the PR (and in the spec today).